src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/metrics/AllMetrics.java

```diff
@@ -42,8 +42,9 @@
     SHARD_STATS,
     MASTER_PENDING,
     MOUNTED_PARTITION_METRICS,
+    CLUSTER_APPLIER_SERVICE,
     ADMISSION_CONTROL_METRICS,
-    SHARD_INDEXING_PRESSURE
+    SHARD_INDEXING_PRESSURE;
```

Comment:
Can we replace ";" with "," to avoid future changes when adding new metric?

Reply:
done